src/vs/workbench/contrib/notebook/browser/contrib/find/notebookFindReplaceWidget.css

```diff
@@ -9,9 +9,9 @@
 	position: absolute;
 	top: -45px;
 	right: 18px;
-	width: 318px;
+	width: var(--notebook-find-width);
 	max-width: calc(100% - 28px - 28px - 8px);
-	pointer-events: none;
```

Comment:
Consulted with @rebornix about this attribute. It can be removed so that the pointer event for the sash occurs.